katacomb/katacomb/conf/parameters/narrow_L.yaml

```diff
@@ -1,21 +1,21 @@
-# Default L-band MFImage configuration options for 'offline' mode
+# Default L-band configuration options for narrowband 'offline' mode
 #
-# These parameters are the defaults used by the `continuum_image.py` script for
+# These parameters are the defaults used by the scripts for
 # offline re-processing. The offline parameters are optimised to produce the
 # best continuum image possible, without consideration for runtime or the
 # quality of CLEAN components for continuum subtraction.
 #
 # Further details of the parameter selection can be found in JIRA ticket
 # SPR1-879: https://skaafrica.atlassian.net/browse/SPR1-879
```

Comment:
This ticket is about the UHF and S-band parameters, not the L narrowband.

Reply:
I've updated the JIRA ticket and as before - I've added the extra tickets you mentioned to this parameter file as well.